test/support.py

```diff
@@ -467,7 +467,11 @@ def _build(tmpdir, ext, hpy_devel, hpy_abi, compiler_verbose=0, debug=None):
 
     # this is the equivalent of passing --hpy-abi from setup.py's command line
     dist.hpy_abi = hpy_abi
+    dist.hpy_no_static_libs = False
```

Comment:
why do you need to set it explicitly here? Isn't it the default?

Reply:
We need to set the attribute here otherwise it just doesn't exist on the `Distribution` object. The tests are not going through `handle_hpy_ext_modules`. This is all done manually here (e.g. we call `fix_distribution`in line 475).